foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -422,3 +422,4 @@ This section contains helpful links to related content. It isn’t required, so
 *   This [W3Schools](https://www.w3schools.com/js/js_htmldom.asp) article offers simple and easy-to-understand lessons on the DOM.
 *   [JS DOM Crash Course](https://www.youtube.com/watch?v=0ik6X4DJKCc&list=PLillGF-RfqbYE6Ik_EuXA2iZFcE082B3s) is an extensive and well explained 4 part video series on the DOM by Traversy Media.
 *   [Understanding The Dom](https://www.digitalocean.com/community/tutorial_series/understanding-the-dom-document-object-model) is an aptly named article-based tutorial series by DigitalOcean.
+*   [Introduction to events - MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events) covers the same topics you learned in this lesson on *events* (videos included).
```

Comment:
I think the "(videos included)" is a little confusing and does add not much value to this sentence. In addition, the use of italics is not needed I prefer moving MDN out of the link. So something like this:

```suggestion
*   [Introduction to events](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events) by MDN covers the same topics you learned in this lesson on events.
```